### PR TITLE
Update sub_wrapup.yml

### PR DESCRIPTION
**Pull request by mosfet80**
update geekyeggo/delete-artifact to v5
v5.0
Switch to actions/artifact, removing the need for a token parameter 
Add default token.
Fix over-arching catch output; errors now correctly result in a failed run  
v4.0
Add support for artifacts uploaded with actions/upload-artifactv4.
Add requirement of token with read and write access to actions.
Update requests to use GitHub REST API.
Deprecate support for actions/upload-artifactv1, actions/upload-artifactv2, and actions/upload-artifact@v3

### CODE REVIEW

**Comment by oursland** [comment]
The v4 has been deprecated in favor of v5.  Please update to the latest.

**Comment by sliptonic** [comment]
revert to draft pending update to v5

File: .github/workflows/sub_wrapup.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ jobs:
         run: |
           mkdir -p ${{ env.artifactsDownloadDir }}
       - name: Download artifacts
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@v5
```
Comment (oursland) [inline .github/workflows/sub_wrapup.yml:57]: This should remain at `v4`.

**Comment by oursland** [comment]
@mosfet80 Please mark your PR as ready for review.

**Comment by oursland** [comment]
@mosfet80 Please mark this PR as ready for review.  [See step 3 of these instructions.](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request)

**Pull request merged**